pkg/proxy/apis/kubeproxyconfig/types.go

```diff
@@ -37,7 +37,7 @@ type ClientConnectionConfiguration struct {
 	// qps controls the number of queries per second allowed for this connection.
 	QPS float32
 	// burst allows extra queries to accumulate when a client is exceeding its rate.
-	Burst int32
```

Comment:
int32 is preferable to int. See https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#primitive-types

> All public integer fields MUST use the Go (u)int32 or Go (u)int64 types, not (u)int (which is ambiguous depending on target platform). Internal types may use (u)int.